litmus-portal/graphql-server/graph/myhub.graphqls

```diff
@@ -0,0 +1,105 @@
+type MyHub {
+	id: ID!
+	GitURL: String!
+	GitBranch: String!
+	IsConfirmed: Boolean!
+	HubName: String!
+}
+
+type Charts {
+	Charts: [Chart!]!
+}
+
+input ChartsInput {
+	HubName: String!
+	UserName: String!
+	RepoOwner: String!
+	RepoBranch: String!
+	RepoName: String!
+}
+
+type Chart {
+	ApiVersion: String!
+	Kind: String!
+	Metadata: Metadata!
+	Spec: Spec!
+	PackageInfo: PackageInformation!
+	Experiments: [Chart!]!
+}
+
+type Maintainer {
+	Name: String!
+	Email: String!
+}
+
+type Link {
+	Name: String!
+	Url: String!
+}
+
+type Metadata {
+	Name: String!
+	Version: String!
+	Annotations: Annotation!
+}
+
+type Annotation {
+	Categories: String!
+	Vendor: String!
+	CreatedAt: String!
+	Repository: String!
+	Support: String!
+	ChartDescription: String!
+}
+
+type Spec {
+	DisplayName: String!
+	CategoryDescription: String!
+	Keywords: [String!]!
+	Maturity: String!
+	Maintainers: [Maintainer!]!
+	MinKubeVersion: String!
+	Provider: String!
+	Links: [Link!]!
+	Experiments: [String!]!
+	ChaosExpCRDLink: String!
+	Platforms: [String!]!
+	ChaosType: String
+}
+
+type Provider {
+	Name: String!
+}
```

Comment:
can be `providerName` if it has just one field.

Reply:
@ishangupta-ds It is required, we need same structure as yamls in chaos-charts, it will be easy to map values in frontend and backend.